symbols_db/handlers/language_handlers/meson_handler.py

```diff
@@ -77,12 +78,14 @@ def find_meson_executables(project_name):
             file_path = Path(root) / file
             if os.access(file_path, os.X_OK):
                 full_path = CWD / file_path
-                file_output = subprocess.run(["file", full_path], capture_output=True)
+                file_output = subprocess.run(
+                    ["file", full_path], capture_output=True, check=False
+                )
                 if b"ELF" in file_output.stdout:
                     executable_list.append(full_path)
     return executable_list
 
 
 def strip_executables(file_path, loc=WRAPDB_LOCATION):
     strip_command = f"strip --strip-all {file_path}".split(" ")
```

Comment:
It's not a good idea to append string like this and use with subprocess.run. These needs to be passed in as args array.